src/pspm_overwrite.m

```diff
@@ -0,0 +1,89 @@
+function ow_final = pspm_overwrite(varargin)
+
+% DESCRIPTION
+% pspm_overwrite generalises the overwriting operation
+% pspm_overwrite considers the following situations
+% - whether options.overwrite is defined
+% - whether PsPM is in develop mode
+% - whether the file exist
+% - whether GUI can be used
+% Only the following situation will stop PsPM to overwrite files
+% 1. if overwrite is defined as not to overwrite
+% 2. if overwrite is not defined, and users use the GUI to stop overwriting
+
+% ARGUMENTS
+% fn        the name of the file to possibly overwrite
+%           can be a link if necessary
+% ow        option of overwrite if this option is presented
+%           can be a value or a struct
+%           if a value, can be 0 (not to overwrite) or 1 (to overwrite)
+%           if a struct, check if the field "overwrite" exist
+
+% OUTPUTS
+% ow_final  option of overwriting determined by pspm_overwrite
+
+pth = fileparts(which('pspm'));
+settings_struct = load([pth, '/pspm_settings.mat'], "settings");
```

Comment:
[minor comment]
Isn't it time-consuming to load the settings like that each time, instead of using a global variable?
It is a genuine question, it might not worth the effort to change this way of doing.